docs/facilitator_tasks.md

```diff
@@ -20,4 +19,4 @@
       * Remind people when the next meeting is and if there is any presentation scheduled for it
 * Get link to video recording from CDF member (usually Dan Lopez), 
 * Ensure the presentation held during the meeting is uploaded to CDF Presentations repo sig-interoperability folder.
-* Commit minutes to the GitHub repo and sending the minutes to the sig-interoperability@lists.cd.foundation (generally Fridays)
\ No newline at end of file
+* Commit minutes to the GitHub repo and sending the minutes to the sig-interoperability@lists.cd.foundation (generally Fridays)
```

Comment:
can we say that the minutes should be committed from HackMD rather than manually from fork so minutes in repo and on HackMD can be kept in sync? Link to HackMD can be included as well.

https://hackmd.io/HuufSDMaTPyb3qxkyBKg3A